lib/hammer_cli_foreman/compute_resource/vmware.rb

```diff
@@ -76,7 +77,8 @@ def provider_vm_specific_fields
             Fields::Field.new(:label => _('Connection Status'), :path => [:connection_status]),
             Fields::Field.new(:label => _('Hardware Version'), :path => [:hardware_version]),
             Fields::Field.new(:label => _('Path'), :path => [:path]),
-            Fields::Field.new(:label => _('Operating System'), :path => [:operatingsystem])
+            Fields::Field.new(:label => _('Operating System'), :path => [:operatingsystem]),
+            Fields::Field.new(:label => _('Boot Order'), :path => [:boot_order])
```

Comment:
It should be just `_('Boot order')`. Although we should change the casing for all of those strings, but it's out of scope for this PR :)